core/src/main/scala/org/apache/spark/executor/CoarseGrainedExecutorBackend.scala

```diff
@@ -123,6 +125,8 @@ private[spark] class CoarseGrainedExecutorBackend(
           executor.stop()
         }
       }.start()
+    case UpdateDelegationTokens(tokens) =>
```

Comment:
Let's add a message that this is received only in mesos case, since CoarseGrainedExecutorBackend is used by both yarn and standalone.